Source/WebCore/page/LocalDOMWindow.cpp

```diff
@@ -2803,4 +2804,9 @@ WebCoreOpaqueRoot root(LocalDOMWindow* window)
     return WebCoreOpaqueRoot { window };
 }
 
+CookieStore* LocalDOMWindow::cookieStore()
+{
+    return nullptr;
```

Comment:
It would be good to lazily construct a CookieStore here, store it as a data member and return a reference to it so that we start to see some tests passing hopefully.

Reply:
Fixed!